GIFrameworkMaps.Web/Program.cs

```diff
@@ -58,19 +62,21 @@ void ConfigureKeyVault(WebApplicationBuilder builder)
                 builder.Configuration.GetSection("KeyVault").GetSection("AzureAd")["DirectoryId"],
                 builder.Configuration.GetSection("KeyVault").GetSection("AzureAd")["ApplicationId"],
                 x509Certificate));
-    }
+
+		builder.Services.Configure<ApiKeyOptions>(builder.Configuration.GetSection(ApiKeyOptions.ApiKeys));
```

Comment:
Whilst we recommend things like the API key are placed in KeyVault, it is not a requirement. As it is written (and correct me if I'm wrong!), if you didn't have KeyVault, the ApiKeyOptions would not get configured, even if you had defined in the appSettings or environment variable. 

For this reason, I think this part should be taken out of the ConfigureKeyVault function and be placed after it (e.g. after the `ConfgureKeyVault(builder)` call). The same could be done to where the GIFrameworkMapsOptions has been configured since someone _could_ put their GIFrameworkMapOptions into KeyVault (not sure why they would, but they might consider some of that data sensitive!)

Reply:
You are correct, the way that I had it assumed that it would be in the Vault, and it would not look to the appsettings.json to populate the ApiKeyOptions. I made a couple of small tweaks to the code to resolve this issue and was able to test it successfully as well. If it looks good to you, I think this can be merged!